### ISSUE TITLE
HPC-GAP: Tasks doc mentions `GAP_STDTASKS`

### ISSUE THREAD

**Issue by ssiccha** [gap-system/gap#2144]
The HPC-GAP manual states:
```
HPC-GAP currently has multiple implementations of the task API. To enable the reference implementation, set the
environment variable <C>GAP_STDTASKS</C> to a non-empty value before starting GAP.
```

This does not seem to be true anymore.

**Comment by fingolfin** [comment]
I think it actually is strill true, but the env var is called `GAP_WORKSTEALING`. But @rbehrends might know better.

**Comment by ssiccha** [comment]
K, I'll remove the commit from #2143 for now.

**Comment by alex-konovalov** [comment]
Resolved in #2143 by correcting the name.
